device-types/Juniper/SRX240H2.yaml

```diff
@@ -7,7 +7,9 @@ front_image: true
 rear_image: true
 weight: 5.1
 weight_unit: kg
-comments: '[Juniper SRX240 Services Gateway Hardware Guide](https://www.juniper.net/documentation/en_US/release-independent/junos/information-products/pathway-pages/hardware/srx240/index.pdf)'
+airflow: right-to-left
+comments: '[Juniper SRX240 Services Gateway Hardware Guide - PDF](https://www.juniper.net/documentation/en_US/release-independent/junos/information-products/pathway-pages/hardware/srx240/index.pdf).
+  Highest supported JUNOS software release 12.3X48-D105 (04 Aug 2020) (only with 2GB RAM). [Upgrade SRX RAM guide](https://michaeldale.com.au/archive/2014/08/23/running-junos-121x47-on-first-gen-srx240h/).'
```

Comment:
I do not believe we should have links to unofficial websites within these files. All links should only be to manufacturer websites.